servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -193,1776 +193,202 @@ void RendererSceneRenderRD::_update_reflection_mipmaps(ReflectionData &rd, int p
 }
 
 void RendererSceneRenderRD::_sdfgi_erase(RenderBuffers *rb) {
```

Comment:
``_sdfgi_erase`` can be removed as ``rb->sdfgi->erase`` can be called directly. 

The functions beginning with  an underscore should be safe to remove altogether as they are not exposed to the API.

We should have a discussion on if we should remove the exposed API functions, move them to rendererSceneForward, or leave them be.

Reply:
Slowly going through these, currently working on the sky stuff which so far is a lot cleaner to move.